### PR TITLE
Update use-feedback-widget.md

### PR DESCRIPTION
**Pull request by iannbing**
Hi, we are releasing [Mendix Feedback Widget](https://appstore.home.mendix.com/link/app/199/) version `8.1.0`. Here's the updated documentation for the new version. Thanks!

### CODE REVIEW

File: content/developerportal/collaborate/use-feedback-widget.md [hunk 1]
```diff
@@ -134,11 +143,17 @@ The feedback feature requires the following properties to be set:
 
 The App Switcher feature requires the [MendixSSO](/developerportal/deploy/mendix-sso) module to be implemented and the following properties to be set:
 
-*  **Advanced** tab > **ID token provider** – the microflow that will return the current ID token via the string value of the decrypted ID token from the MendixSSO module; to obtain the token, the microflow needs to call the **DS_GetCurrentIdToken** microflow from the MendixSSO module:
```
Comment (MarkvanMents) [inline content/developerportal/collaborate/use-feedback-widget.md]: @iannbing 
Will developers using older versions still need to know this information, or have the older versions been removed from the App Store?

File: content/developerportal/collaborate/use-feedback-widget.md [hunk 2]
```diff
@@ -16,7 +16,16 @@ This collaboration is enabled by the [Mendix Feedback Widget](https://appstore.h
 ![](attachments/feedback/widget.png)
 
 {{% alert type="warning" %}}
-This how-to covers the Mendix Feedback Widget [version 6.0.0 and above](#above) as well as [version 5.1.3 and below](#below). There are different usage and configuration steps for these different versions of the widget.
+This how-to covers the Mendix Feedback Widget [version 7.0.0 and above](#above) as well as [version 5.1.3 and below](#below). There are different usage and configuration steps for these different versions of the widget.
+
+If you are trying to upgrade your Mendix Feedback Widget from version 7 to version 8, Studio Pro will not recognize the current configuration of this widget. Add the Mendix Feedback Widget into your page again and configure it accordingly. Then delete the old one.
+
+![](attachments/feedback/cannot-recognize-the-old-version.png)
```
Comment (MarkvanMents) [inline content/developerportal/collaborate/use-feedback-widget.md]: @iannbing 
Is there a reason why, in the image, step 2 comes before step 1? I find it a bit confusing.

Comment (iannbing) [reply content/developerportal/collaborate/use-feedback-widget.md]: They don't necessarily come in order. It depends on where user drag it to. The image is just to explain to the user to remove the one with the red-cross icon.

File: content/developerportal/collaborate/use-feedback-widget.md (hunk 1 above)
Comment (iannbing) [reply content/developerportal/collaborate/use-feedback-widget.md]: Old versions (7.x.x) are still in the App Store.

File: content/developerportal/collaborate/use-feedback-widget.md (hunk 2 above)
Comment (MarkvanMents) [reply content/developerportal/collaborate/use-feedback-widget.md]: OK

**Comment by iannbing** [comment]
Hi @MarkvanMents I added another commit. We intentionally change all the text `version 7.x.x` to `version 8.1.0` to encourage users to use the latest version for better integration.

**Pull request merged**